bety_brapi/controllers_impl/StudiesController_impl.py

```diff
@@ -58,14 +60,14 @@ def studies_study_db_id_get(studyDbId):
             "AND sites.id = experiments_sites.site_id " \
             # "AND experiments.id = " + studyDbId
 
-    print(query)
+    logger.info(query)
 
     if studyDbId:
         query += " AND experiments.id = %s "
         # query = query % studyDbId
         params.append(studyDbId)
 
-    print(query)
+    # print(query)
```

Comment:
use logger.debug when you want to print the query. When we run in production we will run with loglevel INFO.